### PR TITLE
docs - more db pages

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Adds some additional database connection pages:

- Druid
- Presto
- SparkSQL
- SQLite
- SQL Server

### CODE REVIEW

File: docs/api/collection.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 title: "Collection"
 summary: |
   `/api/collection` endpoints. By default, these endpoints operate on Collections in the 'default' namespace, which is
-    the one that has things like Dashboards and Cards. Other namespaces of Collections exist as well, such as the
-    `:snippet` namespace, (called 'Snippet folders' in the UI). These namespaces are completely independent hierarchies.
-    To use these endpoints for other Collections namespaces, you can pass the `?namespace=` parameter (e.g.
-    `?namespace=snippet`).
+  the namespace that has things like Dashboards and Cards. Other namespaces of Collections exist as well, such as the
```
Comment (nllho) [inline docs/api/collection.md:5]: Is this change supposed to be in here? 👀

File: docs/databases/connections/presto.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### Password
+
+The password for the username that you use to connect to the database.
+
+### Use a secure connection (SSL)
+
+Metabase automatically tries to connect to databases with SSL first, then without if that doesn't work. If it's possible to connect to your database with an SSL connection, Metabase will make that the default setting for your database. If you prefer to connect without this layer of security, you can always change this setting later, but we highly recommend keeping SSL turned on to keep your data secure.
+
+### Use SSL certificate?
```
Comment (nllho) [inline docs/databases/connections/presto.md:47]:
```suggestion
```

A rogue heading?

File: docs/databases/connections/presto.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+#### Keystore
```
Comment (nllho) [inline docs/databases/connections/presto.md:49]:
```suggestion
### Keystore
```

File: docs/databases/connections/presto.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+You can specify a local file path, or upload a keystore. You'll also need to input your keystore password.
+
+#### Truststore
```
Comment (nllho) [inline docs/databases/connections/presto.md:53]:
```suggestion
### Truststore
```

File: docs/databases/connections/sql-server.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+The database port. E.g., 1433. To use Dynamic Ports, leave this field empty.
+
+### Database name
+
+The name of the database you're connecting to.
+
+### Database instance name
+
+N/A. If you're running multiple databases on the same host, you can include the instance name here.
```
Comment (nllho) [inline docs/databases/connections/sql-server.md:33]:
```suggestion
If you're running multiple databases on the same host, you can include the instance name here.
```

File: docs/databases/connections/sql-server.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+The password for the username that you use to connect to the database.
+
+### Use a secure connection (SSL)
+
+Metabase automatically tries to connect to databases with SSL first, then without if that doesn't work. If it's possible to connect to your database with an SSL connection, Metabase will make that the default setting for your database. If you prefer to connect without this layer of security, you can always change this setting later, but we highly recommend keeping SSL turned on to keep your data secure.
+
+### ROWCOUNT Override
+
+TODO
```
Comment (nllho) [inline docs/databases/connections/sql-server.md:49]: TODO

**Review by nllho — APPROVED** [review]
A handful of nits and one wild TODO

File: docs/databases/connections/presto.md (hunk 3 above)
Comment (jeff-bruemmer) [reply docs/databases/connections/presto.md:49]: These should be subheadings.

File: docs/api/collection.md (hunk 1 above)
Comment (jeff-bruemmer) [reply docs/api/collection.md:5]: I've updated it upstream in the API code.

**Pull request merged**